src/commands/force/user/create.ts

```diff
@@ -70,20 +71,25 @@ export class UserCreateCommand extends SfdxCommand {
     // because fields is type UserFields & Dictionary<string> we can access these
     const permsets: string = fields.permsets;
     const generatepassword: string = fields.generatepassword;
+    const profileName = fields.profileName;
 
     // extract the fields and then delete, createUser doesn't expect a permsets or generatepassword
```

Comment:
Maybe update from `doesn't expect a permsets or generatepassword` to `doesn't expect convenience fields used locally`